frontend/src/metabase/visualizations/components/Visualization/NoResultsView/NoResultsView.styled.tsx

```diff
@@ -12,7 +12,7 @@ export const Root = styled.div`
   padding-right: 0.5rem;
   padding-bottom: 0.5rem;
 
-  color: var(--mb-color-text-light);
+  color: var(--mb-color-text-tertiary);
```

Comment:
The color for `No results!` text
![image](https://github.com/metabase/metabase/assets/1937582/95c9985d-46ab-4255-b1d7-ca2526d6e3dd)